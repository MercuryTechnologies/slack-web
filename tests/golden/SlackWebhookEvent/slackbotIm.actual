EventEventCallback
    ( EventCallback
        { eventId = EventId
            { unEventId = "Ev044Y3D66M9" }
        , teamId = TeamId
            { unTeamId = "T043DB835ML" }
        , eventTime = MkSystemTime
            { systemSeconds = 1664826477
            , systemNanoseconds = 0
            }
        , event = EventMessage
            ( MessageEvent
                { blocks =
                    [ RichText
                        { blockId = Just
                            ( NonEmptyText "ZRe" )
                        , elements =
                            [ RichTextSectionItemRichText
                                [ RichItemText "You have been removed from #testing-slack-app by "
                                    ( RichStyle
                                        { rsBold = False
                                        , rsItalic = False
                                        }
                                    )
                                , RichItemUser
                                    ( UserId
                                        { unUserId = "U043H11ES4V" }
                                    )
                                    ( RichStyle
                                        { rsBold = False
                                        , rsItalic = False
                                        }
                                    )
                                ]
                            ]
                        }
                    ]
                , channel = ConversationId
                    { unConversationId = "D043HMJ0WDU" }
                , text = "You have been removed from #testing-slack-app by <@U043H11ES4V>"
                , channelType = Im
                , user = UserId
                    { unUserId = "USLACKBOT" }
                , ts = "1664826477.425039"
                , threadTs = Nothing
                , appId = Nothing
                , botId = Nothing
                }
            )
        }
    )