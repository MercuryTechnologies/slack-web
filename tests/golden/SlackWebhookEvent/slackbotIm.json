{
  "token": "aaaa",
  "team_id": "T043DB835ML",
  "api_app_id": "A0442TUPHGR",
  "event": {
    "type": "message",
    "text": "You have been removed from #testing-slack-app by <@U043H11ES4V>",
    "user": "USLACKBOT",
    "ts": "1664826477.425039",
    "team": "T043DB835ML",
    "blocks": [
      {
        "type": "rich_text",
        "block_id": "ZRe",
        "elements": [
          {
            "type": "rich_text_section",
            "elements": [
              {
                "type": "text",
                "text": "You have been removed from #testing-slack-app by "
              },
              {
                "type": "user",
                "user_id": "U043H11ES4V"
              }
            ]
          }
        ]
      }
    ],
    "channel": "D043HMJ0WDU",
    "event_ts": "1664826477.425039",
    "channel_type": "im"
  },
  "type": "event_callback",
  "event_id": "Ev044Y3D66M9",
  "event_time": 1664826477,
  "authorizations": [
    {
      "enterprise_id": null,
      "team_id": "T043DB835ML",
      "user_id": "U0442US8QGH",
      "is_bot": true,
      "is_enterprise_install": false
    }
  ],
  "is_ext_shared_channel": false,
  "event_context": "aaaa"
}
